app/src/main/java/org/simple/clinic/activity/placeholder/PlaceholderScreen.kt

```diff
@@ -5,15 +5,22 @@ import android.util.AttributeSet
 import android.widget.RelativeLayout
```

Comment:
Can you reword the pull request title, please? It should follow the same conventions as our commit messages, i.e, it should be grammatically correct when appended to "If applied, this commit will <PR/commit title>".